docs/joining/application.md

```diff
@@ -0,0 +1,46 @@
+
+The [incubation process](https://github.com/OpenRailAssociation/technical-committee/blob/main/incubation-process.md) defines how projects can become part of the OpenRail Association and what conditions have to be met at which stage. The first step is to apply to be hosted as an official OpenRail Association project. Here is how:
+
+## Decide on adopting an open source model
+
+Open Source provides a lot of benefits: faster innovation, more efficiency, better interoperability. To realize these benefits you need to commit to going with the open source model. This means sharing the code under an open source license, but also to adopt an open governance model, where it's transparent how decisions are taken, and there is a fair opportunity for contribute on eye-level based on the merits of your contributions.
+
+Adopting this model needs some courage. It means giving up central control and adopting a model of collaborative ownership. It means giving up management by status or power and adopting leadership  by quality contributions. It means giving up working behind closed doors and adopting transparent collaboration in public. If you bring up this courage, you will be rewarded by a very effective and enjoyable environment which will deliver results with a speed and quality you might not have seen before.
+
+When you have decided to go the open source way, the OpenRail Association might be the path you choose:
+
+## Check if the project is in the scope of the OpenRail Association
+
+The OpenRail Association facilitates development of open source software in the railway sector. The focus is on railway specific software, which addresses needs of the railway domain. It's not about generic IT infrastructure which is needed by any sector. For these projects other foundations might be a more suitable host. That said, the OpenRail Association is open. If it's about a use case rooted in the railway domain, your project is likely to fit. If in doubt, please ask.
+
+Note that the current focus of the OpenRail Association is open source software, not open data. We realize that open data is an element in a lot of open source projects, especially in the railway domain. So here as well, if in doubt, please ask.
+
+Once you have decided that the OpenRail Association is the right path, go on to the actual application:
+
+## Fill out the questionnaire for new projects
+
+There is a [questionnaire](https://github.com/OpenRailAssociation/technical-committee/blob/main/project-proposals/stage-1/stage-1-questionnaire.md) which helps you to collect the relevant information which is required to describe your project and give the base for the decision of the OpenRail Association on acceptance of your project as an officially hosted OpenRail project.
```

Comment:
Just to be sure: the questionnaire (and incubation process) shall reside just in the repo and not be listed on the website? I somewhat am OK with this, but the downside is that the criteria are not visible directly.

Reply:
I think it's good enough for now. Let's see how we can improve it in the future.